Modified make_match_pannel to accept an arbitrary function (not just IC)

edjuaro opened the pull request.

edjuaro commented:
Feel free to change the order where the parameter function appears, I only put it as third because it seem fitting to my purposes.

KwatME commented:
Thanks!

KwatME commented:
Hi @edjuaro,

I've made some minor styling modifications:

1) Ordering of arguments: function variable is used after target_ascending variable in the code, so I moved the function variable down 1 (now target_ascending is above target):
<img width="597" alt="screen shot 2017-10-19 at 12 31 04" src="https://user-images.githubusercontent.com/7741548/31790291-d905640a-b4c9-11e7-84c1-51bb4bfda986.png">
<img width="725" alt="screen shot 2017-10-19 at 12 31 17" src="https://user-images.githubusercontent.com/7741548/31790382-1ccb80a2-b4ca-11e7-8c06-b244172890d4.png">


2) 80 character limit: I've newline-escaped the argument description that went over 80 characters:
<img width="725" alt="screen shot 2017-10-19 at 12 31 17" src="https://user-images.githubusercontent.com/7741548/31790328-f7dd5ab8-b4c9-11e7-8b3c-2b57a1a117a2.png">

3) Period: I've removed the period at the end of the argument description:
<img width="139" alt="screen shot 2017-10-19 at 12 32 05" src="https://user-images.githubusercontent.com/7741548/31790348-04468aa4-b4ca-11e7-9219-43b591514edc.png">

All looks good and thank you for the pull request!

(I'm going to be propagating the change to ccal and make 0.0.6 if that's ok)

Best,

edjuaro commented:
Yes, that all makes sense to me! Thanks for the attention to detail!

Edwin.

On Thu, Oct 19, 2017 at 12:37 PM, Kwat <notifications@github.com> wrote:

> Hi @edjuaro <https://github.com/edjuaro>,
>
> I've made some minor styling modifications:
>
>    1. Ordering of arguments: function variable is used after
>    target_ascending variable in the code, so I moved the function variable
>    down 1 (now target_ascending is above target):
>
> [image: screen shot 2017-10-19 at 12 31 04]
> <https://user-images.githubusercontent.com/7741548/31790291-d905640a-b4c9-11e7-84c1-51bb4bfda986.png>
>
> [image: screen shot 2017-10-19 at 12 31 17]
> <https://user-images.githubusercontent.com/7741548/31790382-1ccb80a2-b4ca-11e7-8c06-b244172890d4.png>
>
>    1. 80 character limit: I've newline-escaped the argument description
>    that went over 80 characters:
>
> [image: screen shot 2017-10-19 at 12 31 17]
> <https://user-images.githubusercontent.com/7741548/31790328-f7dd5ab8-b4c9-11e7-8b3c-2b57a1a117a2.png>
>
>    1. Period: I've removed the period at the end of the argument
>    description:
>
> [image: screen shot 2017-10-19 at 12 32 05]
> <https://user-images.githubusercontent.com/7741548/31790348-04468aa4-b4ca-11e7-9219-43b591514edc.png>
>
> All looks good and thank you for the pull request!
>
> (I'm going to be propagating the change to ccal and make 0.0.6 if that's
> ok)
>
> Best,
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/KwatME/match/pull/1#issuecomment-338014230>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AVE75u_MI5qt15LoHznSf_Myh-lFlHBrks5st6TugaJpZM4P_o48>
> .
>



-- 
Edwin F. Juárez, PhD
*If you want peace, work for justice. *-Pope Paul VI
